htdocs/compta/facture.php

```diff
@@ -3925,7 +3925,7 @@
 				|| ($object->type == Facture::TYPE_CREDIT_NOTE && empty($discount->id))
 				|| ($object->type == Facture::TYPE_DEPOSIT && empty($discount->id)))
 				&& ($object->statut == 2 || $object->statut == 3 || ($object->statut == 1 && $object->paye == 1))   // Condition ($object->statut == 1 && $object->paye == 1) should not happened but can be found due to corrupted data
-			    && $user->rights->facture->creer)				// A paid invoice (partially or completely)
+			    && $user->rights->facture->reopen)				// A paid invoice (partially or completely)
```

Comment:
The default policy about right is that right to go back is same than right to to the action.
Adding right that differs from that must be done using advanced permission.
So $this->rights[$r][4] = 'reopen'; should be $this->rights[$r][4] = 'reopen_advance';